arb_os/gasAccounting.mini

```diff
@@ -1071,6 +1096,12 @@ view throw func updateCongestionState(state: CongestionState, toBlockNum: uint,
     let gasPoolMax = chainParams_getOrDie(const::Atom_Param_GasPoolMax);
     let speedLimit = chainParams_getOrDie(const::Atom_Param_SpeedLimitPerSecond);
 
+    let rateEstimator = rateEstimator_advanceTime(state.gasUsageRateEstimator, toTimestamp);
+    let rateFactor = rateEstimator_get(rateEstimator) * gasPoolMax / (2 *speedLimit);
```

Comment:
I think we should remove the `2` here

Currently we have the following
```rs
price' = price * (120 * pool_max + factor - pool) / (120 * pool_max)
where factor = min(pool_max, estimate * pool_max / (2 * speed))
```
Suppose the price is rising
```rs
price' = price * (120 * pool_max + factor - pool) / (120 * pool_max)
price' > price 
=> factor > pool 
=> min(pool_max, estimate * pool_max / (2 * speed)) > pool
=> estimate * pool_max / (2 * speed) > pool  // pool_max exceeds pool
```
As we approach the steady state, `estimate` will approximate the speed limit, so
```rs
speed * pool_max / (2 * speed) > pool
pool_max / 2 > pool
=> The pool will be half full at equilibrium
```

Reply:
Having discussed and clarified the 50% is intended to be a short-term buffer, let's finalize a constant (perhaps still 2) and then I'll approve this :)